### ISSUE TITLE
Weird behavior for MediaCount.single

### ISSUE THREAD

**Issue by reliance-engineer** [rafid08/media_picker_widget#7]
Hi again,

When using this configuration:
```
MediaPicker(
                mediaList: const [],
                onPicked: (e) => viewModel.selectMedia(e, context),
                onCancel: () {},
                mediaCount: MediaCount.single,
                mediaType: MediaType.all,
                decoration: PickerDecoration(
                  cancelIcon: const SizedBox.shrink(),
                  completeText: 'Select',
                  completeTextStyle: const TextStyle(
                    color: AppColors.textColorLight,
                  ),
                  completeButtonStyle: ElevatedButton.styleFrom(
                    backgroundColor: AppColors.accent,
                  ),
                ),
              ),
```
when the user clicks on one item it is  selected, but if the user clicks in other item, instead of change the selection to the new item, both Items are selected but only one gets picked

<img src="https://github.com/rafid08/media_picker_widget/assets/114175138/f5dceaf7-da92-4a56-bd2e-255c574cc8c2" width="375" height="667" />

In this image I have the card picture selected, and when I click other picture it gets marked but not selected

**Comment by reliance-engineer** [comment]
Also there is errors in the pub.dev documentation's code:
```
MediaPicker(
  mediaList: mediaList, //let MediaPicker know which medias are already selected by passing the previous mediaList
  onPick: (selectedList){.  /// <-- This is onPicked
    print('Got Media ${selectedList.length}');
  },
  onCancel: ()=> print('Canceled'),
  mediaCount: MediaCount.single,
  mediaType: MediaType.image,
  decoration: PickerDecoration(),
)
```

**Comment by rafidgotit** [comment]
Ah yes! Sorry for missing out the documentation code.

Let me find out the issue in single image picker, also album selector is overlaying your button

**Comment by rafidgotit** [comment]
hey @reliance-engineer ,
It has been solved in v1.0.2.
Thank you for letting me know
